wdio.conf.js

```diff
@@ -0,0 +1,120 @@
+const fs = require('fs-extra')
+const path = require('path')
+const ip = require('ip')
+
+if (!process.env.BROWSERSTACK_USERNAME || !process.env.BROWSERSTACK_ACCESS_KEY) {
+  throw new Error('You must specify both BROWSERSTACK_USERNAME and BROWSERSTACK_ACCESS_KEY')
+}
+
+const TEST_SERVER_HOST = process.env.TEST_SERVER_HOST || ip.address()
+const TEST_SERVER_PORT = process.env.TEST_SERVER_PORT || 3000
+const REPORT_OUTPUT_DIR = './reports/browserstack'
+fs.mkdirpSync(path.resolve(REPORT_OUTPUT_DIR))
+
+exports.config = {
+  user: process.env.BROWSERSTACK_USERNAME,
+  key: process.env.BROWSERSTACK_ACCESS_KEY,
+  services: ['browserstack'],
+  browserstackLocal: true,
+
+  framework: 'jasmine',
+  specs: [
+    './tests/**/*.spec.js',
+  ],
+  suites: {
+    smoke: [
+      './tests/initial-load.spec.js',
+    ],
+  },
+
+  maxInstances: 1,
+  capabilities: [
+    // {
+    //   device: 'iPad 6th',
+    //   real_mobile: true,
+    //   browserName: 'Mobile Safari',
+    // },
+    {
+      os: 'OS X',
+      os_version: 'Mojave',
+      browserName: 'Chrome',
+    },
+    {
+      os: 'OS X',
+      os_version: 'Mojave',
+      browserName: 'Firefox',
+    },
+    {
+      os: 'OS X',
+      os_version: 'Mojave',
+      browserName: 'Opera',
+    },
+    {
+      os: 'OS X',
+      os_version: 'Mojave',
+      browserName: 'Safari',
+    },
+    {
+      os: 'Windows',
+      os_version: '10',
+      browserName: 'Chrome',
+    },
+    {
+      os: 'Windows',
+      os_version: '10',
+      browserName: 'Edge',
+    },
+    {
+      os: 'Windows',
+      os_version: '10',
+      browserName: 'Firefox',
+    },
+    {
+      os: 'Windows',
```

Comment:
is needed to add Opera browser for Win 10

Reply:
BrowserStack does not support automate for Opera on Windows, sorry.